charts/external-dns/README.md

```diff
@@ -16,62 +16,74 @@ After you've installed the repo you can install the chart.
+| `terminationGracePeriodSeconds`   | Termination grace period for the pod.                                                                                                                                                                                                                                                                                  | `null`                                 |
+| `serviceMonitor.enabled`          | If `true`, create a _Prometheus_ service monitor.                                                                                                                                                                                                                                                                      | `false`                                |
+| `serviceMonitor.additionalLabels` | Additional labels to be set on the ServiceMonitor.                                                                                                                                                                                                                                                                     | `{}`                                   |
+| `serviceMonitor.interval`         | _Prometheus_ scrape frequency.                                                                                                                                                                                                                                                                                         | `1m`                                   |
+| `serviceMonitor.scrapeTimeout`    | _Prometheus_ scrape timeout.                                                                                                                                                                                                                                                                                           | `10s`                                  |
+| `env`                             | [Environment variables](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/) for the _external-dns_ container, this supports the full [EnvVar](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core) API including secrets and configmaps.  | `[]`                                   |
+| `namespace`                       | Namespace to install external-dns into. Usefull when installing as a subchart. Defaults to `.Release.Namespace` for backwards compatibility.                                                                                                                                                                           | `""`                                   |
```

Comment:
The only change is the addition of this line. I just reformatted the Markdown table.